src/Core/tests/DeviceTests/Handlers/Editor/EditorHandlerTests.cs

```diff
@@ -515,5 +531,18 @@ public class EditorTextInputTests : TextInputHandlerTests<EditorHandler, EditorS
 			protected override void UpdateCursorStartPosition(EditorHandler editorHandler, int position) =>
 				EditorHandlerTests.UpdateCursorStartPosition(editorHandler, position);
 		}
+
+		class CustomEditorHandler : EditorHandler
+		{
+			// make a copy of the Core mappers because we don't want any Controls changes or to override us
+			public static PropertyMapper<IEditor, IEditorHandler> TestMapper = new(Mapper);
+			public static CommandMapper<IEditor, IEditorHandler> TestCommandMapper = new(CommandMapper);
+
+			// make sure to use our mappers
+			public CustomEditorHandler()
+				: base(TestMapper, TestCommandMapper)
+			{
+			}
+		}
```

Comment:
What's the difference between doing this vs. using ConfigureMauiHandlers as we have with other tests?

i.e. 
```cs
builder.ConfigureMauiHandlers(handler =>
{
	handler.AddHandler<EditorStub, EditorHandler>();
});
```

Reply:
Good point, let me see....

Reply:
> https://github.com/dotnet/maui/blob/95583f7d8527103bd34f58e3000d42b738c35e9e/src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBasement.cs#L141-L143
>
> This means that the `EnsureHandlerCreated` builder is not used for the root view. The builder _is_ used for any children.

Could you elaborate more on this part? I think I'm missing something. Why is it not used for the root view?

Reply:
It _is_ used for the root view. The `EnsureHandlerCreated` is used for the children.

Reply:
After chatting with you offline, I understand now!

to summarize, I learned that:

(in this example, for EntryHandler)
CreateHandler<without type specified>: (which is used implicitly by the specific control pertaining to the tests) doesn't use mappers and therefore we currently add them manually via EnsureHandlerCreated and handler.AddHandler

(in this example, for LayoutHandler)
CreateHandler<THandler>: (which is needed when there are multiple controls different from the control pertaining to the tests) automatically uses mappers and instantiates children (via  InitializeViewHandler --> SetVirtualView --> CreatePlatformElement --> etc) so there's no actual need to manually add mappers here (EnsureHandlerCreated handler.AddHandler stuff)

This is quite confusing though, so @mattleibow has another PR in flux where he is making all the handler mapper test setup stuff a bit more organized and understandable